src/v0/sources/auth0/mapping.json

```diff
@@ -65,6 +65,6 @@
   },
   {
     "sourceKeys": "type",
-    "destKeys": "source_type"
+    "destKeys": ["source_type", "properties.source_type"]
```

Comment:
```suggestion
    "destKeys": "properties.source_type"
```

Reply:
We can remove the root level source_type, Since it will be anyway dropped before exported to warehouse destinations
cc- @krishna2020